src/coordinate_systems.rs

```diff
@@ -9,7 +9,7 @@ use crate::{Coordinate, Vector};
 
 /// Defines how a coordinate system behaves.
 ///
-/// For example, two coordinate systems that implement `CoordinateSystem<NedLike>` both behave
+/// For example, two coordinate systems that implement `CoordinateSystem<Convention = NedLike>` both behave
```

Comment:
Thanks! I think this breaks the expected line length wrapping convention though?

Reply:
Fixed!